Marlin/src/libs/numtostr.h

```diff
@@ -23,7 +23,10 @@
 
 #include <stdint.h>
 
-// Convert a full-range unsigned 8bit int to a percentage
+// Format uint8_t (0-100) as rj string with 123% / _12% / __1% format
+const char* pcttostrpctrj(const uint8_t i);
+
+// Convert uint8_t (0-255) to a percentage, format as above
 const char* ui8tostr4pctrj(const uint8_t i);
```

Comment:
They actually are not the same, The ui8tostrpctrj type convert 0-255 to a percentage. In the Laser/Cutter code you can have RPM, PWM, or Percent. If you use ui8tostrpctrj to display Percent it will convert it incorrectly.

Reply:
Oh, I missed the call inside the argument when I read this:

`return pcttostrpctrj(ui8_to_percent(i));`